src/data/faq_de.json

```diff
@@ -2210,47 +2289,219 @@
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "Siehe <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "Eine zentrale Infrastruktur, welche dazu dient <a href='#glossary_diagnosis_key'>Diagnoseschlüssel</a> über Ländergrenzen hinweg auszutauschen und zu prüfen."

```

Comment:
Add comma after "dient"

in "welche dazu dient"

Reply:
Will be solved in the next commit, thanks!